shared/src/test/diff/nu/MethodSignatures.mls

```diff
@@ -184,11 +184,12 @@ fun f(x: Int): Int
 fun f(Int): Int
 //│ fun f: Int -> Int
 
-// FIXME
+
```

Comment:
You added this FIXME line on a new line in the previous commit. Please remove that new line now.